app/api/srch/shared_params.rb

```diff
@@ -31,5 +31,10 @@ module SharedParams
     params :field do
       optional :field, type: String, documentation: { example: 'username' }
     end
+
+    params :period do
+      optional :from, type: Date, documentation: { example: '14/12/1989' }
```

Comment:
Here, I think @cesswairimu is using a `-` hyphen convention in other date code. Could we switch to that, do you think?

Reply:
Sure! And what validation are you talking about? `search_criteria.validate_period_from_to`?